travis/python_linter.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# usage: 
+#   * set LINT_PYTHON to be the arguments to flake8 (e.g., package name)
+#   * source `python_linter.sh` on Travis CI
+
+pip install flake8
+echo ""
+printf '=%.0s' {1..100}
+echo "Python linter results:"
```

Comment:
```suggestion
printf '=%.0s' {1..40}
echo -n " Python linter results "
printf '=%.0s' {1..40}
echo
```
totally optional

Reply:
<img width="307" alt="image" src="https://user-images.githubusercontent.com/5139267/70759790-5f094200-1cfc-11ea-8a2d-7d6621deb6bf.png">

:( Laziness-inducing message...

Reply:
We'll get this another time